circle.yml

```diff
@@ -1,31 +1,15 @@
 machine:
-  pre:
-    # Install sbt 0.13.16
-    - sudo apt-get install openjdk-8-jdk
-    - wget -q https://dl.bintray.com/sbt/debian/sbt-0.13.16.deb
-    - sudo dpkg -i sbt-0.13.16.deb
-  cache_directories:
-    - "~/.ivy2"
-    - "~/.sbt"
+  services:
+    - docker
 
 dependencies:
-  pre:
-    - sudo apt-get update
-    - sudo apt-get install libffi-dev libssl-dev python-dev
-    - sudo apt-get install python
-    - python --version
-    - curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
-    - sudo python get-pip.py
-    - sudo pip install --upgrade pip
-    - sudo pip install --upgrade setuptools
-    - sudo pip install pyopenssl ndg-httpsclient pyasn1
-    - sudo pip install moto --ignore-installed six
-    - sudo pip install flask
-    - moto_server s3 -p 8001 &
+  override:
+    - docker info
+    - docker build --rm=false -t circleci/moztelemetry:$CIRCLE_SHA1 .
 
 test:
   override:
-    - 'true'
```

Comment:
leave this as true, and put the 'docker run ....' test in the .travis file.  The travis build will run tests and when they pass we merge the PR to master.

Once master receives a change, itll kick off the sbt publish via circleci.

Reply:
If we choose this route, we'll need to keep lines 2-9 so that circleCI can run sbt publish

Reply:
What's the ideal here? It seems odd to have 2 CI systems working together. I think Frank mentioned he'd like it if we eventually moved to CircleCI only? Or maybe I misunderstood.

I started the Docker effort simply to have a nice self-contained environment to work on this project. I don't have the bigger picture. Happy to help flesh that out if there is one?

Reply:
yeah having two ci systems isn't ideal.  Travis is used to run tests on your PR prior to merging.  CircleCI simply run on every commit to master, and publishes the jars to s3/maven

Reply:
In that case should I change this circle.yml file at all? I don't think it's needed in this case. At least not right now for this PR.

Reply:
I would only add tests to circle if we want to migrate off of travis at this moment.  I'm not sure how to set up circleci to run tests on every pr branch

Reply:
Let's take advantage of this and move off travis if we can. @robhudson if you feel like it's extra work you don't want to tackle right now that's fine.

Reply:
though the lines 12-24 that you removed from circle.yml can probably be omitted.  That was my fault for not removing them

Reply:
Ok. My plan will be to keep the docker testing parts and add back the sbt install in the `machine` section so publishing works. Then we can nuke the travisCI and just use CircleCI.